freedomofpress/securedrop
# Update Tor Browser in dev containers

## Status

Ready for review.

## Description of Changes

Fixes #5630.

Changes proposed in this pull request:

* Update Tor Browser to latest stable version in both dev containers.
* Also update Firefox version in Focal container to align with version on which Tor Browser is based.

## Testing

How should the reviewer test this PR?
Write out any special testing steps here.

1. Run `make test-focal`. Ensure Focal dev container build succeeds and all tests pass.
1. Run `make test`. Ensure Xenial dev container build succeeds and all tests pass.

## Deployment

Any special considerations for deployment? Consider both:

1. Upgrading existing production instances.
2. New installs.

## Checklist

### If you made changes to the server application code:


### If you made changes to `securedrop-admin`:


### If you made changes to the system configuration:


### If you made non-trivial code changes:


Choose one of the following:


### If you added or updated a code dependency:

Choose one of the following:
